python/OnnxUdf/example.py

```diff
@@ -0,0 +1,35 @@
+import openeo
+
+# This will be required to import onnxruntime in the UDF.
+dependencies_url = "https://artifactory.vgt.vito.be:443/auxdata-public/openeo/onnx_dependencies.zip"
+# You can upload your own model and paste the url here.
+model_url = "https://artifactory.vgt.vito.be:443/auxdata-public/openeo/test_onnx_model.zip"
+
+spatial_extent = dict(zip(["west", "south", "east", "north"], [8.908, 53.791, 8.96, 54.016]))
+temporal_extent = ["2022-10-01", "2022-12-01"]
+
+connection = openeo.connect("openeo.vito.be").authenticate_oidc()
+
+s2_cube = connection.load_collection(
+    "TERRASCOPE_S2_TOC_V2",
+    temporal_extent=temporal_extent,
+    spatial_extent=spatial_extent,
+    bands=["B04"],
+    max_cloud_cover=20,
+)
+
+udf = openeo.UDF.from_file("onnx_udf.py")
+s2_cube = s2_cube.apply_neighborhood(
+    udf,
+    size=[{"dimension": "x", "value": 256, "unit": "px"}, {"dimension": "y", "value": 256, "unit": "px"}],
+    overlap=[{"dimension": "x", "value": 0, "unit": "px"}, {"dimension": "y", "value": 0, "unit": "px"}],
+)
+
+# We pass the model and dependencies as zip files to the UDF through the job options.
+job_options = {
+    "udf-dependency-archives": [
+        f"{dependencies_url}#tmp/extra_venv",
```

Comment:
Maybe call this "onnx_deps" instead of "extra_venv". I'm also not sure why you put it in a "tmp" folder, everything created by/for the yarn job is temporary anyway.